_episodes/62.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: episode
+title: "62: A lakehouse that simply works at Prezi"
+date: 2024-07-04
+tags: trino prezi
+youtube_id: "update date above to 11th"
+wistia_id: ""
+sections:
+- title: "Introduction "
+  time: 0:00
+- title: Trino releases
+  time:
+- title: Guest Vincenzo Cassaro
+  time:
+- title: Prezi
+  time:
+---
+
+## Hosts
+
+* [Manfred Moser](https://www.linkedin.com/in/manfredmoser), Director of Trino
+  Community Leadership at [Starburst](https://starburst.io),
+  ([@simpligility](https://twitter.com/simpligility))
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+
+## Guest
+
+[Vincenzo Cassaro](https://www.linkedin.com/in/vincenzo-cassaro/) -
+[@viciocassaro](https://twitter.com/viciocassaro), Data Engineer at
+[Prezi](https://prezi.com/)
+
+## Releases and news
+
+[Trino 451](https://trino.io/docs/current/release/release-451.html)
+
+* Add support for configuring a proxy for the S3 native file system.
+* Add `t_pdf` and `t_cdf` functions.
+* Improve performance of certain queries involving window functions.
+* Lots of Iceberg connector improvements including support for incremental
+  refresh for basic materialized views.
+
+[Trino 452](https://trino.io/docs/current/release/release-452.html)
+
+* TBD if release is cut before the 11th .. probably
+
+Other noteworthy topics:
+
+* [Star Poon (oneonestart)](https://github.com/oneonestar) approved as new
+  subproject maintainer for [Trino Gateway](https://trinodb.github.io/trino-gateway/).
+* [Recap blog post]({% post_url 2024-06-24-trino-fest-recap %}) from Trino Fest
+  with video recordings and slides is now available.
+* Trino Contributor Congregation [recap notes](https://github.com/trinodb/trino/wiki/Contributor-meetings#trino-contributor-congregation-14-june-2024) are also available.
+* [Trino Japan meetup](https://techplay.jp/event/944074) happened on the 10th of July.
+
+## Guest Vincenzo Cassaro
+
+Vincenzo has been working with data in all its forms, from data modeling to
+analytics and ML since he completed his masters in computer engineering in
+Italy. He is joining us from there, more specifically from Sicily, to chat with
+us about how he got into computers, learned about Trino, and ended up at Prezi
+now.
+
+## About Prezi
+
+what is Prezi
```

Comment:
Is there a good standard text @vincenzocassaro to use .. or some info from your slide deck